src/style.css

```
@@ -0,0 +1,53 @@
+:root {
+  --background-image: url(./images/star-active.svg);
+}
+
+.body {
+  margin: 0;
+}
+
+.stars {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+
+  display: flex;
+}
+
+.stars__star {
+  display: block;
+  height: 16px;
+  width: 16px;
+
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:first-child {
```

```suggestion
.stars--1 .stars__star:first-child,
.stars--2 .stars__star:nth-child(-n + 2),
.stars--3 .stars__star:nth-child(-n + 3),
.stars--4 .stars__star:nth-child(-n + 4),
.stars--5 .stars__star:nth-child(-n + 5) {
```
Consider grouping selectors to avoid repeating code